install

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+MUJOCO_VERSION=2.1.1
+
+# exit at first error
+set -e
+
+# where config folders will be copied
+GLOBAL_INSTALL=/opt/mpi-is/mujoco # if install called by root
+USER_INSTALL=$HOME/.mpi-is/mujoco # if install called by someone else
+
+# not same installation folder if script called
+# with or without sudo
+if [ "$EUID" -ne 0 ]
+then
+   # without sudo: user home directory
+   INSTALL_DIR=$USER_INSTALL
+   echo "installing configuration file in $INSTALL_DIR"
```

Comment:
This should probably say "installing mujoco" and not "configuration file"?  Same below.

Reply:
indeed